src/main/resources/errors.yaml

```diff
@@ -700,7 +703,7 @@ request-errors:
 
   - scope: SCHEMA
     code: UNSUPPORTED_VECTOR_DIMENSION
-    title: Vector definition contains unsupported configuration
+    title: Vector definition contains unsupported dimension
     body: |-
```

Comment:
Should similarly clarify full message (configuration -> dimension)

Should this also contain name of column for which problem occurs? (maybe follow-up PR)